database/migrations/sqls/20220613072449-update-capture-referenceid-allow-null-up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE capture ALTER COLUMN reference_id NULL;
```

Comment:
ALTER TABLE capture ALTER COLUMN reference_id DROP NOT NULL;

Reply:
Ah, thank you!